### PR TITLE
[KOGITO-1770] - Updating images tests to reflect new examples naming

### PR DESCRIPTION
**Pull request by ricardozanini**
Signed-off-by: Ricardo Zanini <zanini@redhat.com>

See: https://issues.redhat.com/browse/KOGITO-1770

### CODE REVIEW

**Review by radtriste — APPROVED** [review]

**Review by sutaakar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Image location: quay.io/kiegroup/kogito-quarkus-ubi8:latest
 
 For more details have a look at [README.md](modules/kogito-quarkus-ubi8/README.md)
 
-## SpringBoot
+## Spring Boot
```
Comment (spolti) [inline README.md]: Can you please remove it from this PR?
pom will be changed, we can address it here 
https://github.com/kiegroup/kogito-images/pull/134

File: modules/kogito-springboot/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 This S2I is considered a runtime image builder as it relies on another image
 (built with `kogito-springboot-ubi8-s2i`) that actually produced the binaries to be executed
-(the fat jar based on SpringBoot 2.1.x runtime). So this one will only get the binaries
+(the fat jar based on Spring Boot 2.1.x runtime). So this one will only get the binaries
```
Comment (spolti) [inline modules/kogito-springboot/README.md]: same here

**Review by spolti — CHANGES_REQUESTED** [review]

**Comment by spolti** [comment]
are you going to do the same for 0.9.x branch?

**Comment by ricardozanini** [comment]
> are you going to do the same for 0.9.x branch?

Yes, I'm waiting for everyone's approval first to not redo the work in two branches. I'll revert the `SpringBoot` thing. Then please approve so I can send the 0.9.x one.

**Review by spolti — APPROVED** [review]

**Pull request merged**